### DISCUSSION TITLE
Strange Error: What works on dev server does not in Vercel

### DISCUSSION THREAD

**Discussion by shrocket** [neuefische/questions#736] [Web] [answered]
## Good afternoon,

Looking to post our most recent feature in the QA thread, I opened the Vercel deployment and was met by a strange error (please see attached). 

The funny thing is what the title says: The feature works just fine in the dev server. Why on earth should it refuse to work on Vercel?

I'd love for this question to be more detailed, but I'm afraid I might be in over my head on this one. [Here's a link to the relevant PR](https://github.com/ohlunlacassi/hotel-tishjama/pull/10)

Thanks for any help you might be able to provide.

## Your Hotel Tishjama Service Managers

![Screenshot 2023-09-22 at 15 26 02](https://github.com/neuefische/questions/assets/139855611/07d675e7-d9c9-420f-af20-286ddd455eb3)
![Screenshot 2023-09-22 at 15 25 45](https://github.com/neuefische/questions/assets/139855611/4363b071-7e10-47fe-a0ea-4b1d631c7d77)

**Comment by ahohnsen** [comment, marked as answer]
Heyho 👋🏼 

You actually have the same problem as another group had yesterday. Please have a look here: https://github.com/orgs/neuefische/discussions/734
